src/commonMain/kotlin/fr/acinq/lightning/wire/OfferTypes.kt

```diff
@@ -781,12 +781,12 @@ object OfferTypes {
                 nodeParams: NodeParams,
                 trampolineNode: NodeUri,
                 features: Features,
-                pathId: ByteVector32,
```

Comment:
nit: can you update the documentation line 776 accordingly? It should probably be:

```
@param blindingSecret session key used for the blinded path included in the offer.
```

Reply:
Done